stablehlo/transforms/StablehloRefineShapes.cpp

```diff
@@ -359,20 +359,20 @@ struct EvalSliceOpPattern : public OpRewritePattern<SliceOp> {
     if (!resultType.hasRank() || resultType.getRank() != 1)
       return rewriter.notifyMatchFailure(op, "expected 1-dimensional type");
 
-    SmallVector<int64_t> operand;
+    SmallVector<APSInt> operand;
     if (failed(hlo::matchInts(op.getOperand(), operand)))
```

Comment:
Do we have a test that exercises matchInts failing?

Perhaps add an eval slice ui64 test? Can pick from 
https://github.com/openxla/stablehlo/pull/1443/files#diff-803b64030ac18cf321610effd8bb8b56a7b8d52629e683f284bddc597d8a2a7b

Reply:
ui64 isn't going to fail - see tests added in this PR for an example. I'll add an f32 test though, that's a good point.